pkg/activator/throttler_test.go

```diff
@@ -242,18 +278,20 @@ func TestHelper_DeleteBreaker(t *testing.T) {
 	}
 	revID := RevisionID{Namespace: revID.Namespace, Name: revID.Name}
 	throttler.breakers[revID] = queue.NewBreaker(throttler.breakerParams)
-	if len(throttler.breakers) != 1 {
-		t.Errorf("Breaker map size didn't change. Wanted %d, got %d", 1, len(throttler.breakers))
+	if got := len(throttler.breakers); got != 1 {
+		t.Errorf("Breaker map size got %d, want: 1", got)
 	}
-	deleter := DeleteBreaker(throttler)
-	deleter(revision)
+	DeleteBreaker(throttler)(revision)
 	if len(throttler.breakers) != 0 {
 		t.Error("Breaker map is not empty")
 	}
 }
 
-func getThrottler(maxConcurrency int32, revisionGetter func(RevisionID) (*v1alpha12.Revision, error), endpointsGetter func(RevisionID) (int32, error), logger *zap.SugaredLogger, initCapacity int32) *Throttler {
-	params := queue.BreakerParams{QueueDepth: 10, MaxConcurrency: maxConcurrency, InitialCapacity: initCapacity}
+func getThrottler(
+	maxConcurrency int32, revisionGetter func(RevisionID) (*v1alpha12.Revision, error),
+	endpointsGetter func(RevisionID) (int32, error), logger *zap.SugaredLogger,
+	initCapacity int32) *Throttler {
+	params := queue.BreakerParams{QueueDepth: 1, MaxConcurrency: maxConcurrency, InitialCapacity: initCapacity}
```

Comment:
Why the queueDepth change in the params? Should getThrottler just take this as a parameter?

Reply:
The current tests do not call for the depth of more than 1.
And the parameter list is already incessantly long. 
Given these two observations I opted to just change the constant.